components/Autocomplete.tsx

```diff
@@ -0,0 +1,184 @@
+    </Flex>
+  )
+
+  const optionsToDisplay = options.filter(option => {
+    return filterOption
+      ? filterOption(option, search)
+      : (typeof option?.label === 'string' &&
+          option.label.toLowerCase().includes(search?.toLowerCase())) ||
+          (typeof option?.helperText === 'string' &&
+            option.helperText.toLowerCase().includes(search?.toLowerCase()))
+  })
+
+  return (
+    <Popover
+      matchWidth
+      isOpen={isOpen}
+      onOpen={onOpen}
+      onClose={onClose}
+      closeOnBlur={false}
+      closeOnEsc
+      initialFocusRef={inputRef}
+      offset={[0, 0]}
+      placement="bottom"
+    >
+      <PopoverTrigger>
+        <Flex
+          id="Autocomplete"
+          className={bem(...(isOpen ? ['', 'focused'] : []))}
+          tabIndex={0}
+          {...props}
+          onKeyDown={e => e.key === 'Escape' && onClose()}
+          sx={{
+            ...styles?.container,
+            ...props.sx,
+            borderBottomRadius: isOpen ? 0 : undefined,
+            borderBottom: isOpen ? 0 : undefined,
+          }}
+          role="group"
+          justifyContent="center"
+          alignItems="center"
+        >
+          <Flex flexDirection="column" justifyContent="center" w="100%">
+            <Box sx={styles?.label}>{label}</Box>
+            <Box sx={styles?.value} display={val && !isOpen ? 'block' : 'none'}>
+              {val ? (renderSelected || renderDefaultSelected)(val) : null}
+            </Box>
+            <Box sx={styles?.inputWrapper} display={isOpen ? 'block' : 'none'}>
+              <Input
+                {...InputProps}
+                value={search}
+                ref={inputRef}
+                variant="unstyled"
+                onChange={e => {
+                  setSearch(e.target.value)
+                  InputProps?.onChange && InputProps.onChange(e)
+                }}
+              />
+            </Box>
+          </Flex>
+          <Box>
+            <TriangleDownIcon
+              transform={`rotate(${isOpen ? '180deg' : '0'})`}
+              w="10px"
+            />
+          </Box>
+        </Flex>
+      </PopoverTrigger>
+      <PopoverContent w="100%" sx={styles?.popover} ref={popoverRef}>
+        <PopoverBody w="100%">
+          {optionsToDisplay && optionsToDisplay.length > 0 ? (
+            optionsToDisplay.map(option => (
+              <Box
+                w="100%"
+                key={option?.value}
+                className={bem(
+                  'option-wrapper',
+                  val === option ? 'selected' : ''
+                )}
+                sx={styles?.optionWrapper}
+                onClick={() => {
+                  if (val !== option) {
+                    setVal(option)
+                    onSelectOption(option)
+                    onClose()
+                  }
+                }}
+              >
+                {(renderOption || renderDefaultOption)(option)}
```

Comment:
This construct is weird, can we default `renderOption` to `renderDefaultOption` on [47](https://github.com/iron-fish/ui-kit/pull/13/files#diff-1af709b7b4390b50911df9f2d11a10a052d08afa2a527bc35c63933ec7246649R47)?

```
renderOption = renderDefaultOption
```